app/assets/javascripts/webauthn.js

```diff
@@ -28,12 +28,19 @@
     } else {
       response.text().then(function (html) {
         document.body.innerHTML = html;
+        handleClipboard();
```

Comment:
I added the clipboard js here since it should be executed after the success html is set and rendered and not on initial page load.